cli.js

```diff
@@ -45,22 +72,33 @@ function init() {
 	const results = globby.sync([
 		'code_of_conduct.*',
 		'code-of-conduct.*',
+		'CODE_OF_CONDUCT.*',
+		'CODE-OF-CONDUCT.*',
```

Comment:
This is not needed. The globbing is case-insensitive.

Reply:
The reason I turned off case insensitivity and added those cases was for a situation where someone has a COC file in all-uppercase and they want to update it to be the default of lowercase. But I guess it's better if they just `rm` the file first.